README.md

```diff
@@ -2,4 +2,160 @@
 
 [![Build Status](https://travis-ci.org/wayfair/kombucha.svg?branch=master)](https://travis-ci.org/wayfair/kombucha)
 
-A description of this package.
+Smart API snapshot testing and linting
+
+## Overview
+
+Kombucha is a command-line program for performing API snapshot testing. The first time Kombucha runs, it executes the HTTP requests you specify and stores the corresponding JSON responses for those requests. These initial responses become the known-good “snapshots” of the API endpoints you are interested in. 
+
+On subsequent runs, Kombucha compares the live reponse for each endpoint to its corresponding snapshot, and flags differences that it finds problematic. Kombucha is smart enough to know that simple differences in the *values* inside an API response (eg. a value that changed from `true` to `false`, or from `"monday"` to `"tuesday"`) do not constitute breakages of the contract for that API. Instead, Kombucha uses a simple heuristic (for details see [the `structure` check](#the-structure-check)) to determine whether data differences are problematic.
```

Comment:
Typo: `reponse`, `e.g.`
Suggestion: move `(for details see [the `structure` check](#the-structure-check))` to the end of paragraph.